ProcessMaker/Repositories/CaseParticipatedRepository.php

```diff
@@ -17,24 +24,21 @@ class CaseParticipatedRepository
      */
     public function create(CaseStarted $case, TokenInterface $token): void
     {
+        if ($this->checkIfCaseParticipatedExist($token->user->id, $case->case_number)) {
```

Comment:
If the token belongs to a self service task, this line could fail, since `$token->user` will be null for self service

Reply:
Right, same of this [observation](https://github.com/ProcessMaker/processmaker/pull/7421#discussion_r1781654364) the validation is done in the `CaseRepository>updateParticipants` method